t/file.t

```diff
@@ -63,3 +63,13 @@ for my $t (@tests) {
    print "ok $testno\n";
    $testno++;
 }
+
+
+{ #-- https://github.com/libwww-perl/URI/issues/102 -- "5.11 breaks HTML::FormatExternal"
```

Comment:
This could also be 

> Regression test for #102.

Reply:
It should be added, that I was able to validate this test on Linux only.
It can be added to the generic test data structure in the beginning of `t/file.t`
if the results for Windows and MacOs have been verified.